dev/integration_tests/external_ui/test_driver/main_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
it would be nice to have some comments here that document exactly what this is measuring, so that when we come back to it in a few months with some obscure problem, we don't have to first reverse-engineer the test to work out what it's actually testing.

Reply:
Done.